aspnetcore/client-side/libman/samples/LibManSample/libman.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "1.0",
+  "defaultProvider": "cdnjs",
+  "libraries": [
+    {
+      "destination": "wwwroot/lib/jquery/",
+      "files": [
+        "jquery.min.js",
+        "jquery.js",
+        "jquery.min.map"
+      ],
+      "library": "jquery@3.3.1"
+    },
+    {
+      "destination": "wwwroot/lib/bootstrap/",
+      "library": "bootstrap@4.1.3",
+      "provider": "unpkg"
+    },
+    {
+      "destination": "wwwroot/lib/lodash/",
+      "files": [
+        "lodash.js",
+        "lodash.min.js"
+      ],
+      "library": "C:\\tmp\\",
+      "provider": "filesystem"
+    }
+  ]
+}
```

Comment:
CRLF at the end

Reply:
I intentionally didn't do it here, as I display this entire file in the doc. Extra space there drives me bonkers.

Reply:
Does it render? ... I haven't noticed when pulling a full file.

Reply:
It does render, unfortunately. Since comments aren't supported in JSON files, I can't take the approach of displaying a snippet either.